### PR TITLE
[master] Fix memory increase issue of lookups/seed nodes

### PR DESCRIPTION
**Pull request by chetan-zilliqa**
## Description

To fix the continuous increase of memory, clean up the memory cache for STL containers for `seeds` , `lookup`, `ds` and `normal` node.

Lookup, seed = Clear every Tx epoch
DS, Miner node = Clear every vacuous epoch.

## Backward Compatibility

- [ ] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [x] local machine test

- [x] small-scale cloud test

### CODE REVIEW

File: src/libNode/FinalBlockProcessing.cpp [hunk 1]
```diff
@@ -995,6 +1004,15 @@ bool Node::ProcessFinalBlockCore(uint64_t& dsBlockNumber,
       LOG_GENERAL(WARNING, "StoreFinalBlock failed!");
       return false;
     }
+    // Clear STL memory cache
+    if (!LOOKUP_NODE_MODE) {
+      auto clearStlMemoryCache = []() -> void {
+        LOG_GENERAL(INFO, "Clearing STL container cache for mining nodes");
```
Comment (sandipbhoir) [inline src/libNode/FinalBlockProcessing.cpp:1010]: May be we have generic log like - releasing unused memory back to system

**Review by sandipbhoir — CHANGES_REQUESTED** [review]

File: src/libNode/FinalBlockProcessing.cpp (hunk 1 above)
Comment (chetan-zilliqa) [reply src/libNode/FinalBlockProcessing.cpp:1010]: I will remove this log

Comment (chetan-zilliqa) [reply src/libNode/FinalBlockProcessing.cpp:1010]: Also, I will remove the time diff code logic.

File: src/libUtils/CommonUtils.cpp [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#include <malloc.h>
+#include <chrono>
+
+#include "libUtils/Logger.h"
+#include "libUtils/CommonUtils.h"
+
+using namespace std;
+
+void CommonUtils ::ReleaseSTLMemoryCache() {
+  auto startTime = chrono::high_resolution_clock::now();
```
Comment (sandipbhoir) [inline src/libUtils/CommonUtils.cpp:10]: Add `LOG_MARKER()` to this function

**Review by sandipbhoir — CHANGES_REQUESTED** [review]

**Review by ansnunez — APPROVED** [review]

File: src/libUtils/CommonUtils.cpp (hunk 2 above)
Comment (chetan-zilliqa) [reply src/libUtils/CommonUtils.cpp]: done.

**Comment by chetan-zilliqa** [comment]
@sandipbhoir , I have bit refactored the code. Please review the final code.

**Review by sandipbhoir — APPROVED** [review]

**Pull request merged**